backend/fms_core/migrations/0065_v4_12_0.py

```diff
@@ -0,0 +1,66 @@
+    Protocol = apps.get_model("fms_core", "Protocol")
+    StepSpecification = apps.get_model("fms_core", "StepSpecification")
+
+    STEPS = [
+        {"name": "Sample QC (DNA)", "protocol_name": "Sample Quality Control","expected_sample_type": "EXTRACTED_SAMPLE", "specifications": [
+            {"display_name": "Sample QC Type", "sheet_name": "SampleQC", "column_name": "Sample Kind", "value": "DNA"}]
+        },
+        {"name": "Sample QC (RNA)", "protocol_name": "Sample Quality Control","expected_sample_type": "EXTRACTED_SAMPLE", "specifications": [
+            {"display_name": "Sample QC Type", "sheet_name": "SampleQC", "column_name": "Sample Kind", "value": "RNA"}]
+        }
+    ]
+
+    with reversion.create_revision(manage_manually=True):
+        admin_user = User.objects.get(username=ADMIN_USERNAME)
+        reversion.set_comment(f"Create the basic initial workflows.")
+        reversion.set_user(admin_user)
+        #  existing step sample QC to be removed?
+        for step_info in STEPS:
+            protocol = Protocol.objects.get(name=step_info["protocol_name"])
+            step = Step.objects.create(name=step_info["name"],
+                                       protocol_id=protocol.id,
+                                       type="PROTOCOL",
+                                       needs_placement=False,
+                                       needs_planning=False,
+                                       expected_sample_type=step_info["expected_sample_type"],
+                                       created_by_id=admin_user.id,
+                                       updated_by_id=admin_user.id)
+            reversion.add_to_revision(step)
+            for specification in step_info["specifications"]:
+                step_specification = StepSpecification.objects.create(name=specification["display_name"],
+                                                                      sheet_name=specification["sheet_name"],
+                                                                      column_name=specification["column_name"],
+                                                                      value=specification["value"],
+                                                                      step=step,
+                                                                      created_by_id=admin_user.id,
+                                                                      updated_by_id=admin_user.id)
+                reversion.add_to_revision(step_specification)
+        step = Step.objects.get(name="Sample QC")
+        dnaStep = Step.objects.get(name="Sample QC (DNA)")
+        if step:
+            step = Step.objects.get(name="Sample QC")
+            rnaStep = Step.objects.get(name="Sample QC (RNA)")
+            sns = SampleNextStep.objects.filter(step__id=step.id).update(step=rnaStep)
+            so = StepOrder.objects.filter(step__id=step.id).update(step=rnaStep)
+            if sns > 0:
```

Comment:
Odd condition to trigger the deletion of the old step ... once all the changes are complete you should be able to delete without condition. If you want to verify something to make sure there was no issue use an assert so it fails if the condition is not met ... here it would just silently skip the deletion.